01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,34 @@
+const http = require('http')
+
+const server = http.createServer((req, res) => {
+  //   if (req.url === '/') {
+  //     res.end('Welcome to our home page')
+  //   }
+  //   if (req.url === '/about') {
+  //     res.end('Here is our short history')
+  //   }
+  //   res.end(`
+  //   <h1>Oops!</h1>
+  // <p>We can't seem to find the page you are looking for</p>
+  // <a href="/">back home</a>
+  //   `)
+  // ###################################
+  // ###################################
+  //
+  //  IF YOU GET ERRORS WHILE USING ABOVE SETUP,
+  // SWITCH TO IF, ELSE IF, ELSE (BELOW)
+  // WE COVER THE CAUSE, LATER IN EXPRESS TUTORIAL
+  if (req.url === '/') {
+    res.end('Welcome to our home page')
+  } else if (req.url === '/about') {
+    res.end('Here is our short history')
+  } else {
+    res.end(`
+    <h1>Oops!</h1>
+    <p>We can't seem to find the page you are looking for</p>
+    <a href="/">back home</a>
+    `)
```

Comment:
Another extra challenge (optional), if you have some background in frontend, try to change the html for the "not found" page to have something silly, like an image.

Hint: "hotlink" to an image from a third party website if you do that, don't try to get the image to be served by express, unless you feel like figuring that out additionally.